source/localizable/smart_cart/webhooks.html.md.erb

```diff
@@ -7,11 +7,11 @@ title: webhooks
 
 # Webhooks <%= edit_link %>
 
-Smart Cart webhooks allow Skroutz merchants to automatically integrate their
-Smart Cart orders with their internal tools.
+Smart Cart webhooks service allows Skroutz Merchants to **get notified** about new or updated
```

Comment:
There is only one webhook. I think we should rephrase it as:

_Smart Cart webhook allows Skroutz Merchants to automatically receive order details to their platform when a new Smart Cart order is placed or an existing one is updated._